third_party/terraform/tests/resource_pubsub_topic_test.go

```diff
@@ -110,3 +115,31 @@ resource "google_pubsub_topic" "topic" {
 }
 `, pid, topicName, kmsKey)
 }
+
+// Temporary until all destroy functions can be reworked to take a provider as an argument
```

Comment:
is the temporary part the Producer or is it the handwritten destroy?

Reply:
The handwritten destroy. I plan on rewriting the generated destroy to make a destroy check producer that pulls the config from the given provider, similar to this one.